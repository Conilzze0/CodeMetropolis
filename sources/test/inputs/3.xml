<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<buildables from="mapping" to="placing" version="1.0">
    <buildable id="9762a376-6d60-4931-a4f2-e852e8e7d75b" name="" type="container">
        <position x="0" y="0" z="0"/>
        <size x="9" y="9" z="9"/>
        <attributes/>
        <children>
            <buildable id="e3c199c7-a1b1-4057-91ed-caadb084fca6" name="&lt;root_package&gt;" type="ground">
                <position x="0" y="0" z="0"/>
                <size x="9" y="9" z="9"/>
                <attributes/>
                <children>
                    <buildable id="d152f94a-98f0-4ced-ab1b-92cc632d031a" name="unnamed package" type="ground">
                        <position x="0" y="0" z="0"/>
                        <size x="9" y="9" z="9"/>
                        <attributes/>
                        <children>
                            <buildable id="90d8996f-3b9b-4972-9c5c-33af762b543e" name="HelloWorld" type="garden">
                                <position x="0" y="0" z="0"/>
                                <size x="9" y="9" z="9"/>
                                <attributes>
                                    <attribute name="flower-ratio" value="1.0"/>
                                </attributes>
                                <children>
                                    <buildable id="574279d0-3853-4a9e-9ff7-4587d73e7af2" name="void main(String[] args)" type="floor">
                                        <position x="0" y="0" z="0"/>
                                        <size x="9" y="13" z="9"/>
                                        <attributes>
                                            <attribute name="character" value="glass"/>
                                            <attribute name="external_character" value="metal"/>
                                            <attribute name="torches" value="1"/>
                                        </attributes>
                                        <children/>
                                    </buildable>
                                </children>
                            </buildable>
                        </children>
                    </buildable>
                </children>
            </buildable>
        </children>
    </buildable>
</buildables>
