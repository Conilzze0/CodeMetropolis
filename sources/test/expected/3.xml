<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<buildables from="placing" to="rendering" version="1.0">
    <buildable id="5ffac7cc-083c-44c6-8eae-a80bdbf46b93" name="" type="container">
        <position x="0" y="60" z="0"/>
        <size x="33" y="9" z="33"/>
        <attributes/>
        <children>
            <buildable id="db16ce5c-8b86-4ce4-a526-cf79c5b49751" name="&lt;root_package&gt;" type="ground">
                <position x="3" y="61" z="3"/>
                <size x="27" y="9" z="27"/>
                <attributes/>
                <children>
                    <buildable id="5853c970-bdf4-41a8-acb9-b3e8e9a22b66" name="unnamed package" type="ground">
                        <position x="6" y="62" z="6"/>
                        <size x="21" y="9" z="21"/>
                        <attributes/>
                        <children>
                            <buildable id="38f2dd25-aa08-436a-8bb6-2925291adada" name="HelloWorld" type="garden">
                                <position x="9" y="63" z="9"/>
                                <size x="15" y="9" z="15"/>
                                <attributes>
                                    <attribute name="flower-ratio" value="1.0"/>
                                </attributes>
                                <children>
                                    <buildable id="7e69cfbb-f22c-4f99-b806-6441caf82a3e" name="void main(String[] args)" type="floor">
                                        <position x="12" y="64" z="12"/>
                                        <size x="9" y="13" z="9"/>
                                        <attributes>
                                            <attribute name="character" value="glass"/>
                                            <attribute name="external_character" value="metal"/>
                                            <attribute name="torches" value="1"/>
                                        </attributes>
                                        <children/>
                                    </buildable>
                                </children>
                            </buildable>
                        </children>
                    </buildable>
                </children>
            </buildable>
        </children>
    </buildable>
</buildables>
